games/devtest/mods/benchmarks/init.lua

```diff
@@ -114,4 +114,39 @@ minetest.register_chatcommand("bench_bulk_set_node", {
 	end,
 })
 
+minetest.register_chatcommand("bench_bulk_get_node", {
+	params = "",
+	description = "Benchmark: Bulk-get 99×99×99 stone nodes",
+	func = function(name, param)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			return false, "No player."
+		end
+		local pos_list = {}
+		local ppos = player:get_pos()
+		local i = 1
+		for x=2,100 do
+			for y=2,100 do
+				for z=2,100 do
+					pos_list[i] = {x=ppos.x + x,y = ppos.y + y,z = ppos.z + z}
```

Comment:
copy-paste, please move to function to deduplicate

Reply:
done